cms/envs/common.py

```diff
@@ -994,6 +994,9 @@
 XBLOCK_SETTINGS = {
     "VideoDescriptor": {
         "licensing_enabled": FEATURES.get("LICENSING", False)
+    },
+    'VideoModule': {
+        'YOUTUBE_API_KEY': YOUTUBE_API_KEY
```

Comment:
I don't know how we should handle this, but clearly the API needs to work for developers on devstack, on sandboxes, in BokChoy etc. It looks like we need to create an application key, but that if we published it to GitHub it would get hijacked:

https://developers.google.com/youtube/registering_an_application

Maybe instead you could have to write up a document describing how developers can get their own key and configure their devstack to use it. Ideally it would be a script that they run so that it isn't too complex to do. 

Note that I do think that sandboxes should be configured correctly by default, so you'd have to communicate with devops to work out how to handle this. In fact, they are probably the right folks to recommend how we handle this even on devstack. @e0d, do you have any recommendations here?